src/main/java/ladder/view/InputView.java

```diff
@@ -4,23 +4,25 @@
 
 public class InputView {
 
+  private static final String separator = System.lineSeparator();
```

Comment:
```suggestion
  private static final String SEPARATOR = System.lineSeparator();
```

static 상수는 언제나 대문자로 네이밍하는 습관을 들이시면 좋을 것 같습니다.
java 컨벤션이라서 소소하게 언급해봅니다.